Sources/SwiftGodot/Core/GArray.swift

```diff
@@ -41,3 +41,23 @@ extension GArray: Collection {
         }
     }
 }
+
+extension GArray {
+    public convenience init<T: VariantStorable>(_ type: T.Type = T.self) {
+       self.init(
+			base: GArray(),
+			type: Int32(T.Representable.godotType.rawValue),
+			className: T.Representable.godotType == .object ? StringName("\(T.self)") : StringName(),
+			script: Variant()
+		)
+    }
+
+    public convenience init<T: VariantStorable>(_ array: [T]) {
+        self.init(T.self)
+        array.forEach { append(value: Variant($0)) }
+    }
+	
+	public func asArray<T: VariantStorable>(_ type: T.Type = T.self) -> [T] {
```

Comment:
helper unwrapper. wondering if it should return either empty or the full array, filtering out the untyped elements by using compactMap seems a bit odd. maybe it should throw?